pkg/collector/corechecks/containers/docker.go

```diff
@@ -392,7 +394,9 @@ func (d *DockerCheck) reportIOMetrics(io *cmetrics.ContainerIOStats, tags []stri
 	// Write values per device, or fallback to sum
 	if len(io.DeviceWriteBytes) > 0 {
 		for dev, value := range io.DeviceWriteBytes {
-			sender.Rate("docker.io.write_bytes", float64(value), "", append(tags, "device:"+dev))
+			devTags := append(tags, "device:"+dev)
```

Comment:
Nit: you could have been with just adding the new field to the append call:

`append(tags, "device:"+dev, "device_name:"+dev)`